rpmrc.in

```diff
@@ -461,6 +472,8 @@ arch_compat: armv5tl: armv4tl
 arch_compat: armv4tl: armv4l
 arch_compat: armv4l: armv3l
 arch_compat: armv3l: noarch
+arch_compat: armv8hnl: armv8hl
+arch_compat: armv8hl: armv7hnl
```

Comment:
This is not compatible. This does not describe a true superset to subset case.

Reply:
Since NEON is mandatory on aarch64, armv8hnl and armv8hl are the same thing, so it does describe a true superset to subset case. (To be even more correct, should probably add arch_compat: armv8hl: armv8hnl -- but I don't think circular loops there are a good idea.)

It may be worth arguing armv8hnl shouldn't exist at all (because armv8hl is the same and therefore the extra n is superfluous) or armv8hl shouldn't exist at all (because armv8hnl is the same and having the n in there is consistent with what was done on armv7) -- I've put in both for now to be compatible with what either side would expect.